Modules/Microsoft365DSC/Examples/Resources/PPTenantSettings/1-ConfigurePPTenantSettings.ps1

```diff
@@ -0,0 +1,36 @@
+<#
+This example sets Power Platform tenant settings.
+#>
+
+Configuration Example
+{
+  param(
+    [Parameter(Mandatory = $true)]
+    [PSCredential]
+    $credsGlobalAdmin
+  )
+  Import-DscResource -ModuleName Microsoft365DSC
+
+  node localhost
+  {
+    PPTenantSettings TenantSettings
+    {
+      WalkMeOptOut                                   = $false
```

Comment:
Missing the mandatory IsSingleInstance parameter

Reply:
Fixed in new commit